pkgs/tools/security/proxmark3/proxmark3-rrg.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, mkDerivation, stdenv, fetchFromGitHub, pkgconfig, gcc-arm-embedded
+, bluez5, qt5, readline
```

Comment:
qt5 should not be inputted directly but through libsForQt5.callPackage in top level.